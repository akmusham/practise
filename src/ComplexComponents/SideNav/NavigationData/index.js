module.exports = [
  {
    url: "speed",
icon: `data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAyMi4wLjEsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiDQoJIHZpZXdCb3g9IjAgMCA1MTIgNTEyIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA1MTIgNTEyOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPg0KCS5zdDB7ZmlsbDojMDJDRTlEO30NCgkuc3Qxe2ZpbGw6I0JDQkVDMDt9DQo8L3N0eWxlPg0KPGc+DQoJPGc+DQoJCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0yMCwyMzljLTUuNywwLTExLjMtMi40LTE1LjMtNy4xYy03LjEtOC40LTYtMjEuMSwyLjQtMjguMmwyMzYtMTk5YzcuNS02LjMsMTguMy02LjMsMjUuOCwwbDIzNiwxOTkNCgkJCWM4LjQsNy4xLDkuNSwxOS43LDIuNCwyOC4yYy03LjEsOC40LTE5LjcsOS41LTI4LjIsMi40TDI1Niw0Ni4xTDMyLjksMjM0LjNDMjkuMSwyMzcuNCwyNC42LDIzOSwyMCwyMzl6Ii8+DQoJPC9nPg0KCTxnPg0KCQk8cGF0aCBjbGFzcz0ic3QwIiBkPSJNNDQwLDUxMkgzMjguNWMtMTEsMC0yMC05LTIwLTIwVjM4NmMwLTI4LjktMjMuNi01Mi41LTUyLjUtNTIuNXMtNTIuNSwyMy42LTUyLjUsNTIuNXYxMDZjMCwxMS05LDIwLTIwLDIwDQoJCQlINzJjLTExLDAtMjAtOS0yMC0yMFYyNzUuNWMwLTExLDktMjAsMjAtMjBzMjAsOSwyMCwyMFY0NzJoNzEuNXYtODZjMC01MSw0MS41LTkyLjUsOTIuNS05Mi41czkyLjUsNDEuNSw5Mi41LDkyLjV2ODZINDIwVjI3NS41DQoJCQljMC0xMSw5LTIwLDIwLTIwczIwLDksMjAsMjBWNDkyQzQ2MCw1MDMuMSw0NTEsNTEyLDQ0MCw1MTJ6Ii8+DQoJPC9nPg0KPC9nPg0KPC9zdmc+DQo=`,
    navTitle: "Home",
    backgroundColor: "transparent"
  },
  {
    url: "bot",
    icon: `data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAyMi4wLjEsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiDQoJIHZpZXdCb3g9IjAgMCA1MTIgNTEyIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA1MTIgNTEyOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPg0KCS5zdDB7ZmlsbDojMDJDRTlEO30NCgkuc3Qxe2ZpbGw6I0JDQkVDMDt9DQo8L3N0eWxlPg0KPGc+DQoJPGc+DQoJCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0yNTYsNTEyYy02OC40LDAtMTMyLjctMjYuNi0xODEtNzVDMjYuNiwzODguNywwLDMyNC40LDAsMjU2UzI2LjYsMTIzLjMsNzUsNzVDMTIzLjMsMjYuNiwxODcuNiwwLDI1NiwwDQoJCQlzMTMyLjcsMjYuNiwxODEsNzVjNDguNCw0OC40LDc1LDExMi42LDc1LDE4MXMtMjYuNiwxMzIuNy03NSwxODFDMzg4LjcsNDg1LjQsMzI0LjQsNTEyLDI1Niw1MTJ6IE0yNTYsMzkuOA0KCQkJYy0xMTkuMiwwLTIxNi4yLDk3LTIxNi4yLDIxNi4yczk3LDIxNi4yLDIxNi4yLDIxNi4yYzExOS4yLDAsMjE2LjItOTcsMjE2LjItMjE2LjJTMzc1LjIsMzkuOCwyNTYsMzkuOHoiLz4NCgk8L2c+DQoJPGc+DQoJCTxnPg0KCQkJPHBhdGggY2xhc3M9InN0MCIgZD0iTTM1Ni4xLDMzOS40Yy0xMSwwLTE5LjktOC45LTE5LjktMTkuOVYxOTIuNWMwLTExLDguOS0xOS45LDE5LjktMTkuOXMxOS45LDguOSwxOS45LDE5Ljl2MTI2LjkNCgkJCQlDMzc2LDMzMC41LDM2Ny4xLDMzOS40LDM1Ni4xLDMzOS40eiIvPg0KCQk8L2c+DQoJCTxnPg0KCQkJPHBhdGggY2xhc3M9InN0MCIgZD0iTTE1NS45LDMzOS40Yy0xMSwwLTE5LjktOC45LTE5LjktMTkuOVYxOTIuNWMwLTExLDguOS0xOS45LDE5LjktMTkuOXMxOS45LDguOSwxOS45LDE5Ljl2MTI2LjkNCgkJCQlDMTc1LjgsMzMwLjUsMTY2LjksMzM5LjQsMTU1LjksMzM5LjR6Ii8+DQoJCTwvZz4NCgk8L2c+DQo8L2c+DQo8L3N2Zz4NCg==`,

    navTitle: "Bots",
    backgroundColor: "transparent"
  },
  {
    url: "skill",
    icon: "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAyMi4wLjEsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiDQoJIHZpZXdCb3g9IjAgMCA1MTIgNTEyIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA1MTIgNTEyOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPg0KCS5zdDB7ZmlsbDojMDJDRTlEO30NCgkuc3Qxe2ZpbGw6I0JDQkVDMDt9DQo8L3N0eWxlPg0KPGc+DQoJPHBhdGggY2xhc3M9InN0MCIgZD0iTTM4Mi42LDUxMkgyMC4yQzksNTEyLTAuMSw1MDIuOS0wLjEsNDkxLjhWMzY0LjZjMC03LjUsNC4yLTE0LjQsMTAuOS0xNy45YzYuNy0zLjUsMTQuNy0zLDIwLjksMS4zDQoJCWM3LjYsNS4zLDE2LjYsOC4xLDI1LjksOC4xYzI1LjEsMCw0NS41LTIwLjQsNDUuNS00NS41YzAtMjUuMS0yMC40LTQ1LjUtNDUuNS00NS41Yy05LjMsMC0xOC4yLDIuOC0yNS45LDguMQ0KCQljLTYuMiw0LjMtMTQuMyw0LjgtMjAuOSwxLjNDNC4xLDI3MS0wLjEsMjY0LjEtMC4xLDI1Ni42VjEyOS40YzAtMTEuMiw5LjEtMjAuMiwyMC4zLTIwLjJoOTguNGMtMi4xLTcuNS0zLjItMTUuMy0zLjItMjMuMg0KCQljMC00Ny40LDM4LjYtODYsODYtODZzODYsMzguNiw4Niw4NmMwLDcuOS0xLjEsMTUuNy0zLjIsMjMuMmg5OC40YzExLjIsMCwyMC4zLDkuMSwyMC4zLDIwLjJ2OTguNGM3LjUtMi4xLDE1LjMtMy4yLDIzLjMtMy4yDQoJCWM0Ny40LDAsODYsMzguNiw4Niw4NnMtMzguNiw4Ni04Niw4NmMtNy45LDAtMTUuNy0xLjEtMjMuMy0zLjJ2OTguNEM0MDIuOCw1MDIuOSwzOTMuOCw1MTIsMzgyLjYsNTEyeiBNNDAuNCw0NzEuNWgzMjEuOVYzNTkuOA0KCQljMC04LDQuNy0xNS4yLDExLjktMTguNWM3LjMtMy4zLDE1LjgtMiwyMS43LDMuM2M4LjMsNy4zLDE5LDExLjQsMzAuMSwxMS40YzI1LjEsMCw0NS41LTIwLjQsNDUuNS00NS41DQoJCWMwLTI1LjEtMjAuNC00NS41LTQ1LjUtNDUuNWMtMTEuMSwwLTIxLjgsNC0zMC4xLDExLjRjLTYsNS4zLTE0LjUsNi42LTIxLjcsMy4zYy03LjMtMy4zLTExLjktMTAuNS0xMS45LTE4LjVWMTQ5LjdIMjUwLjcNCgkJYy04LDAtMTUuMi00LjctMTguNS0xMS45Yy0zLjMtNy4zLTItMTUuOCwzLjMtMjEuN2M3LjMtOC4zLDExLjQtMTksMTEuNC0zMC4xYzAtMjUuMS0yMC40LTQ1LjUtNDUuNS00NS41UzE1NS45LDYwLjksMTU1LjksODYNCgkJYzAsMTEuMSw0LDIxLjgsMTEuNCwzMC4xYzUuMyw2LDYuNiwxNC41LDMuMywyMS43Yy0zLjMsNy4zLTEwLjUsMTEuOS0xOC41LDExLjlINDAuNHY3Ni43YzUuNi0xLjEsMTEuMy0xLjcsMTcuMi0xLjcNCgkJYzQ3LjQsMCw4NiwzOC42LDg2LDg2cy0zOC42LDg2LTg2LDg2Yy01LjgsMC0xMS42LTAuNi0xNy4yLTEuN1Y0NzEuNXoiLz4NCjwvZz4NCjwvc3ZnPg0K",
    navTitle: "Skills",
    backgroundColor: "transparent"
  },
  // {
  //   url: "app",
  //   icon:
  //     "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyCAYAAAAeP4ixAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAGwSURBVGhD7ZQ9SMNQFIU71c3Jyd3N0UXwZ2hilQquTaKIL2mqODiUgoMKbgoOToI/CIKDlE4uDi5WN0fByanSTXAQXH2+Z9+Ud0l9zx+u9R74KNyc5J6TJsmQSCQSiYRBTpENuX50KKh1xAsP3Fk2qE7Fo3Ev7nO86EWE5J8nfJoMgl51CRwSJfIynPh9mPCjuBPiH2lJf85jw+oSOJQrsoK6yzdqlCpR5K5dPB5RIxyiIj9ZxPXZGPRcpyFCbTpeeO0Wozp0PIn4wp1/+H22Bh3vRD6IRlVcWK4XVdt3Fj+OH1ZUbF2iyD10EkpEVhVbl3h2m+BJGBFZVWxdXV9kemGZz5RWNOQc8ttgvMOmSLy+xVd3jzTkHPLbYLwjrUhhbv52KmCPSZY2tl+hJXIO+W0w3SGzqti63q6yTd7o4UnOTsraAomcQ34bTHfIrCq2LiryDfxKkcVKCXzh5Bzy22C6438XOT0u8+rOvoacQ34bTHdYFemad+T5sp+3LgY05Bzy22C6w6oIRqgINqgINqgINqgINqgINtKLNLJ7wlT7C8isKjaJRCKRSF9UJvMO8tZxXOYdZHEAAAAASUVORK5CYII=",
  //   navTitle: "Apps"
  // }
  // {
  //   url: "setting",
  //   icon: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyCAMAAAAp4XiDAAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAAJcEhZcwAADsMAAA7DAcdvqGQAAABFUExURV98i1lzgV98i198i158i1t3hl98i0dwTFpxgVp3hV98i0RZYl98i198i0RaY0RZYl9+jF98i158i0VbZVp2hFVue09ncuGccvoAAAATdFJOU/417KC7aswACBVO8Ifh1XQn1d2JLeU0AAACFUlEQVRIDcWV24LqIAxF6TVQtVZb/f9PHUiyQzp6jp15GR5sBRbsXBvoxyP8mKC/Q5aux2je6H4nbAg2uoPI2YjQH0S6itwOIn1F2oPIWJEQX5lqfnXOxSEJSGOwIV24DDw7+UvC3SZHwECacnJ7TskZIpeNTRzu5XVQBgh2trJv2x6PbZP3oHOtilRk0lV+bM9VxxMUz2tcBYneYgMK9/RniYcEcfHecAOe7iLxACNLPeqFWFfHsAcYcRmCs/2zHsgeKEisczs7gDl7prydb7Fsd7LmGYST1pdgM0KNegyXnK45Bul6UgrXSDAFoSjm6JaT5lsDhpXfFgm/IkQlmNB1lTWiq55RnNYhLw2hPPvQHZa+SSceebFYzsOQVJEZi0Tqg4IgK2sf+wVSggNhUE3xv8I+mn+GXLXliJP7Xb00XHYh1FBmbfF7KC8u+S33EZh1Pc0z4ugSxjKZ6z5bnweuUbPlgXzJ68gxX5O7vfqHD+Mf7rdsvqv8Ks1gV2JL8Zp4rK8HvTCOED8L4qz5Zo+zI2hXEoRcKeeENifsm5JmmSJJ2xsE5s6Xex/+yRMdVhHiWs4NdtKg1t3jsogG9Hkg8aYdOw67C++sJp3bYN9AIJSshJZ6Q7BPUpwsvQ1Bnu56VID+uoq4+Bkip+zo5/VWlZkB7tA3wsjlgtXVB8TF1XrEB6QZMfrFbcXrO2FY+8fzF8gXgtJIC9GwTDIAAAAASUVORK5CYII=",
  //   navTitle: "Settings"
  // }
]
