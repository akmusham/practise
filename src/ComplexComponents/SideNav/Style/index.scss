.navContainer {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  border: 1px solid #e2e2e2;
  background-color: #ffffff;
  width: 5%;

  .nav-child-container {
    height: 100%;
    width: 70%;
    padding: 1em 0.6em 1em 0.5em;
    display: flex;
    flex-direction: column;

    :last-child {
      margin-top: auto;
    }
  }
}
