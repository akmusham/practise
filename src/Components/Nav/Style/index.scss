.nav {
  display: flex;
    flex-flow: column;
    align-items: center;
        margin-bottom: 0.5em;
  span {
    font-size: 1em;
    margin-top: 0.5em;
    font-family: 'Roboto', sans-serif;
  }
}
.navIcon {
  display: flex;
  flex-flow: column;
  img {
    width: 100%;
    height: auto;
  }
}
.navIcon:hover {
  cursor: pointer;
}
