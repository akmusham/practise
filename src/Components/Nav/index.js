import React from "react"
import ReactHtmlParser, { processNodes, convertNodeToElement, htmlparser2 } from 'react-html-parser';
import { Link } from "react-router-dom"
import AvatarIcon from "../AvatarIcon"

import "./Style/index.scss"

export default class Nav extends React.Component {
  constructor(props) {
    super(props)
    this.state = {}
  }

  render() {
    return (
      <div className="nav">
        <Link to={`/user/${this.props.url}`}>
          <div className="navIcon">
            <img src={this.props.icon} style={{width:"100%"}} />
          </div>
        </Link>
        <span>{this.props.navTitle}</span>
      </div>
    )
  }
}
