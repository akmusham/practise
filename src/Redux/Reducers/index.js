import { combineReducers } from "redux"
import { routerReducer } from "react-router-redux"
import { loadingBarReducer } from 'react-redux-loading-bar'
import loginOperations from "./LoginReducers"
import registrationOperations from "./RegistrationReducers"
import chatWindowOperations from "./ChatWindowReducers"
import skillOperations from "./SkillReducers"
import botOperations from "./BotReducers"
import notificationOperations from "./NotificationReducers"
import logoutOperations from "./LogoutReducers"
import mschatOperations from "./MsChatReducers"
import newTeamOperations from "./NewTeamReducer"
import backchannelOperations from "./BackchannelReducers"
import resetPasswordOperations from "./ResetPasswordReducers"
import dashboardWidgetOperations from "./DashboardWidgetReducers"
import mystackOperations from "./MyStackReducer"
import agileDashboardOperations from "./AgileDashboardReducer/ProjectData.json"
import listTeamOperations from "./ListTeamReducers"
import listMemTeamOperations from "./ListMemTeamReducers"
import inviteUserOperations from "./InviteUserReducers"
import addMemTeamOperations from "./AddMemberTeamReducer"
import deleteTeamOperations from "./DeleteTeamReducer"
import editTeamOperations from "./EditTeamReducers"

export default combineReducers({
  loadingBar: loadingBarReducer,
  registrationOperations: registrationOperations,
  loginOperations: loginOperations,
  chatWindowOperations: chatWindowOperations,
  skillOperations: skillOperations,
  botOperations: botOperations,
  notificationOperations: notificationOperations,
  routing: routerReducer,
  logoutOperations: logoutOperations,
  newTeamOperations: newTeamOperations,
  mschatOperations: mschatOperations,
  backchannelOperations: backchannelOperations,
  resetPasswordOperations: resetPasswordOperations,
  dashboardWidgetOperations: dashboardWidgetOperations,
  mystackOperations: mystackOperations,
  agileDashboardData:agileDashboardOperations,
  listTeamOperations: listTeamOperations,
  listMemTeamOperations:listMemTeamOperations,
  inviteUserOperations:inviteUserOperations,
  addMemTeamOperations:addMemTeamOperations,
  deleteTeamOperations:deleteTeamOperations,
  editTeamOperations:editTeamOperations
})
