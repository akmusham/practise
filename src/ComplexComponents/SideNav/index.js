import React from "react"
import MuiThemeProvider from "material-ui/styles/MuiThemeProvider"
import Divider from "material-ui/Divider"
import IconMenu from "material-ui/IconMenu"
import MenuItem from "material-ui/MenuItem"
import IconButton from "material-ui/IconButton"
import Download from "material-ui/svg-icons/file/file-download"
import MoreVertIcon from "material-ui/svg-icons/navigation/more-vert"
import ArrowDropRight from "material-ui/svg-icons/navigation-arrow-drop-right"
import Icon from "../../Components/Icon"
import Nav from "../../Components/Nav"
import Card from "../../Components/Card"
import navProps from "./NavigationData"
import SettingsMenu from "./Components/SettingsMenu"
import ChatButton from "./Components/ChatButton"
import FeedbackButton from "./Components/FeedbackButton"
import NodeRedButton from "./Components/NodeRedButton"

import "./Style/index.scss"

export default class SideNav extends React.Component {
  constructor(props) {
    super(props)
    this.state = {}
  }

  render() {
    return (
      <div className="navContainer">
        <div className="nav-child-container">
          {navProps.map((obj, index) => <Nav {...obj} key={index} />)}
          <div className="nav">
            <div className="navIcon">
              <ChatButton />
              <span>Chat</span>
            </div>
          </div>
          <div className="nav">
            <div className="navIcon">
              <FeedbackButton />
              <span>Feedback</span>
            </div>
          </div>
          <div className="nav">
            <div className="navIcon">
              <SettingsMenu />
              <span>Profile</span>
            </div>
          </div>
        </div>
      </div>
    )
  }
}
