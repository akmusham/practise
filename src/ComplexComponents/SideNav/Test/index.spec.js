import React from "react"
import { ConnectedRouter } from "react-router-redux"
import { expect } from "chai"
import { mount } from "enzyme"
import sinon from "sinon"
import { Provider } from "react-redux"
import renderer from "react-test-renderer"
import store from "../../../../../Redux/Store"
import SideNav from "../"
import Nav from "../../../../../Components/Nav"
import navProps from "../NavigationData"
import { history } from "../../../../../Redux/Store"

describe("render connected SideNav component", () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(
      <Provider store={store}>
        <ConnectedRouter history={history}>
          <SideNav />
        </ConnectedRouter>
      </Provider>
    )
  })

  it("should render component", () => {
    expect(wrapper.length).to.equal(1)
  })

  it("should check if props are passed properly", () => {
    expect(wrapper.find(Nav)).to.have.length(navProps.length)
  })

  it("should match snapshot", () => {
    const renderedValue = renderer.create(
      <Provider store={store}>
        <ConnectedRouter history={history}>
          <SideNav />
        </ConnectedRouter>
      </Provider>
    )
    expect(renderedValue.toJSON()).to.matchSnapshot()
  })
})
