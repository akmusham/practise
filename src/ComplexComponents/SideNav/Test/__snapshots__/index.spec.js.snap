// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render connected SideNav component should match snapshot 1`] = `
<div>
  <div>
    <div
      className="navContainer"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": "0px",
          "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        }
      }
    >
      <div
        className="nav"
      >
        <a
          href="/user/home"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color="#000"
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#fff",
                  "borderRadius": "50%",
                  "color": "#000",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-pied-piper-pp"
                  color="#000"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#000",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          
        </span>
      </div>
      <div
        className="nav"
      >
        <a
          href="/user/speed"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color={undefined}
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgb(188, 188, 188)",
                  "borderRadius": "50%",
                  "color": "#ffffff",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-tachometer"
                  color="#ffffff"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          Speed
        </span>
      </div>
      <div
        className="nav"
      >
        <a
          href="/user/bot"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color={undefined}
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgb(188, 188, 188)",
                  "borderRadius": "50%",
                  "color": "#ffffff",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-rocket"
                  color="#ffffff"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          Bots
        </span>
      </div>
      <div
        className="nav"
      >
        <a
          href="/user/skill"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color={undefined}
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgb(188, 188, 188)",
                  "borderRadius": "50%",
                  "color": "#ffffff",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-th-large"
                  color="#ffffff"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          Skills
        </span>
      </div>
      <div
        className="nav"
      >
        <a
          href="/user/project"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color={undefined}
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgb(188, 188, 188)",
                  "borderRadius": "50%",
                  "color": "#ffffff",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-folder"
                  color="#ffffff"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          Projects
        </span>
      </div>
      <div
        className="nav"
      >
        <a
          href="/user/app"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color={undefined}
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgb(188, 188, 188)",
                  "borderRadius": "50%",
                  "color": "#ffffff",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-file"
                  color="#ffffff"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          Apps
        </span>
      </div>
      <div
        className="nav"
      >
        <a
          href="/user/setting"
          onClick={[Function]}
        >
          <div>
            <div
              className="navIcon"
              color={undefined}
              size={35}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgb(188, 188, 188)",
                  "borderRadius": "50%",
                  "color": "#ffffff",
                  "display": "inline-flex",
                  "fontSize": 17.5,
                  "height": 35,
                  "justifyContent": "center",
                  "muiPrepared": true,
                  "userSelect": "none",
                  "width": 35,
                }
              }
            >
              <div>
                <span
                  className="fa fa-cog"
                  color="#ffffff"
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "color": "#ffffff",
                      "display": "inline-block",
                      "fontSize": 24,
                      "muiPrepared": true,
                      "position": "relative",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "userSelect": "none",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </a>
        <span
          className="navTitle"
        >
          Settings
        </span>
      </div>
    </div>
  </div>
</div>
`;
